variables.tf

```diff
@@ -43,21 +46,55 @@ variable "gitlab_address_name" {
 ##########################
 
 variable "postgresql_version" {
+  type        = string
   description = "(Required) The PostgreSQL version to use. Supported values for Gitlab POSTGRES_12, POSTGRES_13. Default: POSTGRES_12"
   default     = "POSTGRES_12"
 }
 
 variable "postgresql_tier" {
+  type        = string
   description = "(Required) The machine type to use.Postgres supports only shared-core machine types, and custom machine types such as db-custom-2-13312"
   default     = "db-custom-2-8192"
 }
 
+variable "postgresql_disk_size" {
+  type        = number
+  description = "he size of data disk, in GB. Size of a running instance cannot be reduced but can be increased. Default to 100 GB"
+  default     = "100"
+}
+
+variable "postgresql_disk_type" {
+  type        = string
+  description = "The type of postgresql data disk: PD_SSD or PD_HDD. "
+  default     = "PD_SSD"
+}
+
+variable "postgresql_availability_type" {
+  type        = string
+  description = "The availability type of the Cloud SQL instance, high availability (REGIONAL) or single zone (ZONAL)."
+  default     = "ZONAL"
+}
+
+variable "postgresql_location" {
+  type        = string
+  description = "The preferred compute engine zone."
+  default     = "europe-west1-b"
```

Comment:
we should stick with a default zone/region for the all services if not defined, there are very few cases where you want services dislocated in different zones

Reply:
just to be clear we should default to a global region/zone defined for the project

Reply:
You mean the entire module, in its integrity, should stay in a single region/zone. Is that right?